index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
```

Comment:
- [ ] [OPTIONAL] I would suggest adding a link to your css file in your index.html `<head>` so that your styling applies to your web page to make it look aesthetically pleasing ✨ 💖 😄